js/services/MeteorService.ts

```diff
@@ -36,6 +38,4 @@ const buildMeteorsDataResponse = (nearEarthObjects, meteorDto) => {
             were_dangerous: meteorDto.wereDangerous ? wereDangerous : undefined
```

Comment:
The same question about complex object

Reply:
What values can be in the 'meteorDto.wereDangerous'? number, string, undefined, etc
Usually, a special function is used to remove fields with undefined values